src/email.cc

```diff
@@ -44,16 +44,9 @@
 #include <regex>
 
 Smtp::Smtp()
-    : _host{}
```

Comment:
In a general way, it is better to do constructor initialization than do a dedicated initialize method, mainly when you have only one constructor, except if you have a well expressed rational (do you have one here ?).
Moreover, since C++11, you can even put these default settings directly in the class declaration as default assigned values. It make them publicly readable from the header (if shared).